frontend/components/LiveQuery/TargetsInput/_styles.scss

```diff
@@ -48,16 +48,14 @@
       box-shadow: 0px 4px 10px rgba(52, 59, 96, 0.15);
       box-sizing: border-box;
 
-      &__inner {
```

Comment:
Why were we able to unwrap this?

Reply:
It was just nested divs, felt overkill to style only the most child div